packages/ra-ui-materialui/src/layout/CardContentInner.tsx

```diff
@@ -40,10 +41,18 @@ const CardContentInner = props => {
     );
 };
 
+export declare type CardContentInnerClassKey = 'root';
+
+export interface CardContentInnerProps {
+    className?: string;
+    classes?: ClassNameMap<CardContentInnerClassKey>;
```

Comment:
We prefer another approach to typing classes, see https://github.com/mui-org/material-ui/issues/17973#issuecomment-639281445